### ISSUE TITLE
Flaky MonoDevelop.PackageManagement test

### ISSUE THREAD

**Issue by Therzok** [mono/monodevelop#3683]
## Steps to Reproduce

1. https://devdiv.visualstudio.com/DevDiv/_build/index?buildId=1293642
2. Happened on another PR too

### On which platforms did you notice this

[x] macOS
[ ] Linux
[ ] Windows

> VS bug #571171

**Comment by mrward** [comment]
Not sure why this is failing on VSTS. It has never failed on Wrench.

```
1) ReadPackages_SearchTextAndIncludePrereleaseIsTrue_SearchTextAndIncludePrereleaseUsedWhenQueryingPackageFeed (MonoDevelop.PackageManagement.Tests.AllPackagesViewModelTests.ReadPackages_SearchTextAndIncludePrereleaseIsTrue_SearchTextAndIncludePrereleaseUsedWhenQueryingPackageFeed)
2018-01-17T14:37:44.5227960Z      Expected string length 4 but was 0. Strings differ at index 0.
2018-01-17T14:37:44.5242270Z   Expected: "Test"
2018-01-17T14:37:44.5256150Z   But was:  <string.Empty>
2018-01-17T14:37:44.5274060Z   -----------^
2018-01-17T14:37:44.5285160Z 
2018-01-17T14:37:44.5301700Z   at MonoDevelop.PackageManagement.Tests.AllPackagesViewModelTests+<ReadPackages_SearchTextAndIncludePrereleaseIsTrue_SearchTextAndIncludePrereleaseUsedWhenQueryingPackageFeed>d__26.MoveNext () [0x000f9] in /Users/vsts/agent/2.126.0/work/1/s/monodevelop/main/src/addins/MonoDevelop.PackageManagement/MonoDevelop.PackageManagement.Tests/MonoDevelop.PackageManagement.Tests/AllPackagesViewModelTests.cs:297 
2018-01-17T14:37:44.5312550Z 
```
Are the VSTS tests being run in parallel?

**Comment by Therzok** [comment]
It simply does `make test assembly=$ASSEMBLY`, so it doesn't run in parallel.

**Comment by mrward** [comment]
Just saw this test fail on wrench. First time that has happened.

```
1) ReadPackages_SearchTextAndIncludePrereleaseIsTrue_SearchTextAndIncludePrereleaseUsedWhenQueryingPackageFeed (MonoDevelop.PackageManagement.Tests.AllPackagesViewModelTests.ReadPackages_SearchTextAndIncludePrereleaseIsTrue_SearchTextAndIncludePrereleaseUsedWhenQueryingPackageFeed)
     Expected string length 4 but was 0. Strings differ at index 0.
  Expected: "Test"
  But was:  <string.Empty>
  -----------^

  at MonoDevelop.PackageManagement.Tests.AllPackagesViewModelTests+<ReadPackages_SearchTextAndIncludePrereleaseIsTrue_SearchTextAndIncludePrereleaseUsedWhenQueryingPackageFeed>d__26.MoveNext () [0x000c8] in /Users/builder/data/lanes/432/ecd14eae/source/monodevelop/main/src/addins/MonoDevelop.PackageManagement/MonoDevelop.PackageManagement.Tests/MonoDevelop.PackageManagement.Tests/AllPackagesViewModelTests.cs:297 
```

Still not sure why this fails.
